gamera.lua

```diff
@@ -0,0 +1,205 @@
+-- gamera.lua v1.0.1
```

Comment:
Not specifically related to this MR, but perhaps we should keep external libraries in a separate directory (`classic.lua` and `gamera.lua` in this case), that way it's a clearer picture. Maybe call the folder `util/`?